src/tests/functional/plugin/shared/src/behavior/ov_plugin/properties_tests.cpp

```diff
@@ -74,16 +74,19 @@ void OVSetPropComplieModleGetPropTests::SetUp() {
 }
 
 TEST_P(OVEmptyPropertiesTests, SetEmptyProperties) {
+    core->get_versions(target_device);
```

Comment:
why do we need it here? `get_property` already creates plugin

Reply:
yes, thank you, I missed it about get_property